tests/by-util/test_cksum.rs

```diff
@@ -35,6 +35,7 @@ fn test_empty() {
 }
 
 #[test]
+#[ignore]
```

Comment:
this is the one I asked about in discord (my nick is Ody there). Did you also look at why it fails?

Reply:
nope :)
Do you want to give it a try?

Reply:
https://github.com/uutils/coreutils/pull/2054